R/plotScoreHeatmap.R

```diff
@@ -410,11 +414,21 @@ plotScoreHeatmap <- function(results, cells.use = NULL, labels.use = NULL,
         }
     }
 
-    # Trim by labels (max.labels) & normalize
-    # Determine labels to show based on 'max.labels' with the highest
-    # pre-normalized scores, normalize, then remove other labels.
-    maxs <- rowMaxs(scale(t(scores)))
-    to.keep <- order(maxs,decreasing=TRUE)[seq_len(max.labels)]
+    ## Trim by labels (max.labels) & normalize
+    # Determine labels to show based on 'max.labels' with the highest...
+    if (sum(is.na(scores)) < 0.10*prod(dim(scores))) {
+    	# (Individual reference, should really have no NAs)
```

Comment:
Four space indenting.

Reply:
Sorry for that sloppiness. I blame an Rstudio update, but I've found the preference that should have caused this and it shouldn't happen again.